src/Sets/Interval.jl

```diff
@@ -633,3 +633,53 @@ function rectify(x::Interval{N}) where {N<:Real}
         return Interval(zero(N), max(x.dat.hi, zero(N)))
     end
 end
+
+"""
+    diameter(x::Interval, [p]::Real=Inf)
+
+Compute the diameter of an interval.
+
+### Input
+
+- `x` -- interval
+- `p` -- (optional, default: `Inf`) norm
+
+### Output
+
+A real number representing the diameter.
+
+### Notes
+
+In one dimension all norms are equivalent.
+"""
+function diameter(x::Interval, p::Real=Inf)
+    return norm(max(x) - min(x), p)
+end
+
+"""
+    split(x::Interval, num_blocks)
+
+Partition an interval into uniform sub-intervals.
+
+### Input
+
+- `x`          -- interval
+- `num_blocks` -- number of blocks, possibly wrapped in a vector
```

Comment:
perhaps say that here "block" means "sub-interval". not sure that "block" is standard though, i woud just use `k` ..

Reply:
I copied the name `num_blocks` from the `AbstractHyperrectangle` method. But I agree that in this case we should just call it `k` because it is not a keyword argument.